File: 프로그래머스/1/42840. 모의고사/모의고사.py
def solution(answers):
    answer = []
    one = [1,2,3,4,5];
    two = [2,1,2,3,2,4,2,5]
    three = [3,3,1,1,2,2,4,4,5,5]
    
    count = [0,0,0];
    for i in range(len(answers)) :

        if answers[i] == one[i%5] :
            count[0] += 1;
        if answers[i] == two[i%8] :
            count[1] += 1;
        if answers[i] == three[i%10] :
            count[2] += 1;
        
    for i in range(len(count)) :
        if max(count) == count[i] :
            answer.append(i+1)
    answer.sort()
    
    return answer

File: 프로그래머스/1/42840. 모의고사/모의고사.js
function solution(answers) {
    let answer=[]
    var score = [0,0,0];
    let one = [1,2,3,4,5]
    let two = [2,1,2,3,2,4,2,5]
    let three = [3,3,1,1,2,2,4,4,5,5]
    
    let len = answers.length
    for(let i = 0; i < len; i++){
        if(answers[i] == one[i%one.length]){
            score[0] += 1
        }
        if(answers[i] == two[i%two.length]){
            score[1] += 1
        }
        if(answers[i] == three[i%three.length]){
            score[2] += 1
        }
    }
    for(let i = 0;i<score.length ; i++){
        if(Math.max(...score) == score[i]){
            answer.push(i+1)
        }
    }
    return answer;
}